ERROR 2024-10-11 11:21:56,639 log Internal Server Error: /internship/
Traceback (most recent call last):
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Priyesh Pandey\OneDrive\Desktop\agratas\AIPL_SOURCE_CODE\AIPL_WEB_ERP\AIPL\views.py", line 438, in internship_opportunity_page
    return render(request, 'Internship/internship.html')
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\backends\django.py", line 62, in render
    return self.template.render(context)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 958, in render_annotated
    return self.render(context)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\templatetags\static.py", line 116, in render
    url = self.url(context)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\templatetags\static.py", line 113, in url
    return self.handle_simple(path)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\templatetags\static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\contrib\staticfiles\storage.py", line 166, in url
    return self._url(self.stored_name, name, force)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\contrib\staticfiles\storage.py", line 145, in _url
    hashed_name = hashed_name_func(*args)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\contrib\staticfiles\storage.py", line 465, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'img/internship_background.jpg'
ERROR 2024-10-11 11:21:56,641 basehttp "GET /internship/ HTTP/1.1" 500 145
ERROR 2024-10-16 14:36:38,685 log Internal Server Error: /internship/
Traceback (most recent call last):
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Priyesh Pandey\OneDrive\Desktop\agratas\AIPL_SOURCE_CODE\AIPL_WEB_ERP\AIPL\views.py", line 483, in internship_opportunity_page
    return render(request, 'Internship/internship.html')
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\engine.py", line 176, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\engine.py", line 158, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 14: 'static'. Did you forget to register or load this tag?
ERROR 2024-10-16 14:36:38,685 basehttp "GET /internship/ HTTP/1.1" 500 147730
ERROR 2024-10-16 14:40:47,406 log Internal Server Error: /internship/
Traceback (most recent call last):
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Priyesh Pandey\OneDrive\Desktop\agratas\AIPL_SOURCE_CODE\AIPL_WEB_ERP\AIPL\views.py", line 483, in internship_opportunity_page
    return render(request, 'Internship/internship.html')
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\engine.py", line 176, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\engine.py", line 158, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\Priyesh Pandey\AppData\Roaming\Python\Python310\site-packages\django\template\base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 14: 'static'. Did you forget to register or load this tag?
ERROR 2024-10-16 14:40:47,406 basehttp "GET /internship/ HTTP/1.1" 500 146986
